msal/tests/Test.MSAL.NET.Unit.net45/CoreTests/InstanceTests/B2cAuthorityTests.cs

```diff
@@ -76,34 +76,7 @@ public void NotEnoughPathSegmentsTest()
                 Assert.AreEqual(CoreErrorMessages.B2cAuthorityUriInvalidPath, exc.Message);
             }
         }
-
-        [TestMethod]
-        [TestCategory("B2CAuthorityTests")]
-        public void ValidationEnabledNotSupportedTest()
```

Comment:
>public void ValidationEnabledNotSupportedTest() [](start = 8, length = 47)

you should probably keep a test saying even if you set it to true, no authority validation is happening?

Reply:
added 2 tests, one for b2clogin.com and one for login.microsoftonline.com w/the json response for openid w/validate=true.

---
In reply to: [234388511](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1405#discussion_r234388511) [](ancestors = 234388511)